### DISCUSSION TITLE
Acer Chromebppk Spin 514, Crosh and sudo

### DISCUSSION THREAD

**Discussion by NazarychUA** [skycocker/chromebrew#9477] [Q&A]
crosh> shell
Sudo commands will not succeed by default.
If you want to use sudo commands, use the VT-2 shell
(Ctrl-Alt-{F2/Right arrow/Refresh}) or build the image with the
login_enable_crosh_sudo USE flag:

$ USE=login_enable_crosh_sudo emerge-$BOARD chromeos-login
or
$ USE=login_enable_crosh_sudo cros build-packages --board=$BOARD
    
chronos@ezkinil-rev4 / $ sudo chromeos-setdevpasswd
sudo crossystem dev_boot_signed_only=1
sudo: The "no new privileges" flag is set, which prevents sudo from running as root.
sudo: If sudo is running in a container, you may need to adjust the container configuration to disable the flag.
sudo: The "no new privileges" flag is set, which prevents sudo from running as root.
sudo: If sudo is running in a container, you may need to adjust the container configuration to disable the flag.

**Comment by satmandu** [comment]
Is there a question?

You need to run `sudo` commands from VT-2, which you get to using: `(Ctrl-Alt-{F2/Right arrow/Refresh})`
